python/sglang/srt/openai_api/adapter.py

```diff
@@ -380,7 +402,7 @@ def v1_generate_request(all_requests):
         else:
             prompt_kwargs = {"input_ids": prompt}
     else:
-        if isinstance(prompts[0], str) or isinstance(propmt[0][0], str):
+        if isinstance(prompts[0], str):
```

Comment:
why is this different from L400?
```
# L400
if isinstance(prompt, str) or isinstance(prompt[0], str):

# L405
if isinstance(prompts[0], str):
```

Reply:
because currently we do not support parallel sampling/ list input in Batch API so I just remove the second condition check